ui/jobs/types.d.ts

```diff
@@ -0,0 +1,48 @@
+import { BaseOptions } from '../../types/data';
+import { Job } from '../../types/job';
+import { BuffInfo } from './buff_tracker';
+
+export interface IDefaultOptions extends BaseOptions {
+  JustBuffTracker: boolean;
+  LowerOpacityOutOfCombat: boolean;
+  OpacityOutOfCombat: number;
+  PlayCountdownSound: boolean;
+  HideWellFedAboveSeconds: number;
+  ShowMPTickerOutOfCombat: boolean;
+  MidHealthThresholdPercent: number;
+  LowHealthThresholdPercent: number;
+  BigBuffShowCooldownSeconds: number;
+  BigBuffIconWidth: number;
+  BigBuffIconHeight: number;
+  BigBuffBarHeight: number;
+  BigBuffTextHeight: number;
+  BigBuffBorderSize: number;
+  GpAlarmPoint: number;
+  GpAlarmSoundVolume: number;
+}
+
+export interface IInitOptions {
+  ShowHPNumber: Job[];
+  ShowMPNumber: Job[];
+  ShowMPTicker: Job[];
+
+  MaxLevel: number;
+
+  PerBuffOptions: {
+    [s: string]: Partial<BuffInfo>;
+  };
+
+  FarThresholdOffence: number;
+  PldMediumMPThreshold: number;
+  PldLowMPThreshold: number;
+  DrkMediumMPThreshold: number;
+  DrkLowMPThreshold: number;
+  /**  One more fire IV and then despair. */
+  BlmMediumMPThreshold: number;
+  /** Should cast despair. */
+  BlmLowMPThreshold: number;
+}
+
+export type IJobsOptions = IDefaultOptions & IInitOptions;
```

Comment:
I'm not sure I understand the `I` prefix here.  Is this Hungarian/Microsoft naming? I'd maybe suggest:

* `IJobsOptions` => `JobsOptions`
* `IDefaultOptions` => `DefaultOptions` or `NonConfigOptions`
* `IInitOptions` => `InitOptions` or `ConfigOptions`

Reply:
Yep, the `I` prefix means `Interface`, which is widely used in Microsoft's code. But I think it is good to change as you suggested.